lib/segment/src/entry/entry_point.rs

```diff
@@ -147,6 +145,21 @@ pub trait SegmentEntry {
     /// Number of deleted points
     fn deleted_point_count(&self) -> usize;
 
+    /// Size of all available vectors in storage
+    fn available_vectors_size_in_bytes(&self, vector_name: &str) -> OperationResult<usize>;
+
+    /// Max value from all `available_vectors_size_in_bytes`
+    fn max_available_vectors_size_in_bytes(&self) -> OperationResult<usize> {
+        let mut max_size = 0;
+        for vector_name in self.vector_names() {
+            max_size = std::cmp::max(
+                max_size,
+                self.available_vectors_size_in_bytes(&vector_name)?,
+            );
+        }
+        Ok(max_size)
+    }
```

Comment:
We can simplify this a bit with [`max`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.max).

Reply:
Thanks for fixing it!